docs/application/native/tutorials/process/setting-properties.md

```diff
@@ -72,9 +74,9 @@ You can perform the following tasks using the **Overview** tab:
   You can add a launcher icon to your application by defining it in the **Icon** panel. If your application supports both HD and WVGA, it is best to provide an icon for HD for better image quality in both cases.
 
   The following table describes the available icons.
-  ​
+
   | Profile  | Mandatory | Format                 | Size              | Size              | Description                        |
```

Comment:
line 78: Add the Table title.
line 202: Code tag was not applied all table. 
line 209: <name> is not shown in MD. 
line 225: Code tag was not applied all table.

Reply:
line 78: fixed.
line 202: The comment about code tag is same as above.
The others were fixed.